src/components/AddSection/AddSection.component.js

```diff
@@ -0,0 +1,23 @@
+import React,{useRef} from 'react';
+import './addsection.styles.css'
+
+function AddSection({children}) {
+
+  const containerRef = useRef()
+
+  const inject = (e) => {
+    e.preventDefault()
+    containerRef.current.innerHTML= containerRef.current.innerHTML + containerRef.current.innerHTML
+  }
+
+  return (
+    <div>
+      <button className="buttonx" onClick={inject} >+ add</button>
+      <div ref={containerRef} >
+        {children}
+      </div>
+    </div>
+  )
```

Comment:
This is good, but a better way to handle this would be to use state. 
state = { sections: [{ ... }] }